### PR TITLE
chore: Prepare for release 4.13.1-liferay.4

### PR DESCRIPTION
**Pull request by julien**

### CODE REVIEW

File: patches/0001-LPS-89596-Cannot-Drag-Image-from-top-content-line-in.patch [hunk 1]
```diff
@@ -25,5 +25,5 @@ index aa32f0de5..289684816 100644
  				return;
  
 -- 
-2.25.0
+2.17.1
```
Comment (wincent) [inline patches/0001-LPS-89596-Cannot-Drag-Image-from-top-content-line-in.patch]: You really should upgrade 😂

**Comment by wincent** [comment]
@julien: Just a thought which I should have had when reviewing this. In the future we should merge one PR to add the patch and *then* do the release. Because otherwise we wind up with a changelog that has an entry in it like "chore: Prepare for release 4.13.1-liferay.4" and no mention of the actually interesting thing, which is that we fixed a bug.

**Comment by julien** [comment]
@wincent yeah that makes sense, thanks.

File: patches/0001-LPS-89596-Cannot-Drag-Image-from-top-content-line-in.patch (hunk 1 above)
Comment (julien) [reply patches/0001-LPS-89596-Cannot-Drag-Image-from-top-content-line-in.patch]: image
:tada:

**Pull request merged**